eng/build.yml

```diff
@@ -20,7 +20,7 @@ parameters:
 phases:
 - template: /eng/common/templates/job/job.yml
   parameters:
-    name: ${{ parameters.agentOs }}
+    name: ${{ parameters.agentOs }}${{ parameters.pgoInstrument }}
```

Comment:
This makes the name of the job `Windows_NTtrue`, which I think is unintentional.